app/views/admin/poll/questions/_form.html.erb

```diff
@@ -1,18 +1,34 @@
+<%= render "admin/shared/globalize_locales", resource: @question %>
+
 <%= form_for(@question, url: form_url) do |f| %>
 
   <%= render 'shared/errors', resource: @question %>
 
+	<% @question.globalize_locales.each do |locale| %>
+    <%= hidden_field_tag "delete_translations[#{locale}]", 0 %>
+	<% end %>
+
   <%= f.hidden_field :proposal_id %>
 
   <div class="small-12">
     <div class="small-12 medium-6 large-4">
+      <% select_options = Poll.all.map { |p| [p.name, p.id] } %>
```

Comment:
We will use `Poll.all.map...` instead of `Poll.pluck` cause we want to make use of [the fallbacks set for i18n](https://github.com/consul/consul/blob/master/config/application.rb#L24) for missing translations

Reply:
:thinking: I think [this comment](https://github.com/consul/consul/pull/2833#discussion_r215298180) can also be applied in here. What do you think, @microweb10 ?

Reply:
Not the same at all. 
Here we don't iterate over (`each`). 
We actually need to `map`, which involves storing in memory (`array`) the values 😊